readme.md

```diff
@@ -30,6 +31,14 @@ pod 'MapboxDirections.swift', :git => 'https://github.com/mapbox/MapboxDirection
 pod 'MapboxNavigation.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :tag => 'v0.0.4'
 ```
 
+#### [Carthage](https://github.com/carthage/carthage)
+
+Add the following line to your Cartfile:
+
+```ruby
+github "mapbox/MapboxNavigation.swift" "swift3"
```

Comment:
Oh, I copied from the Cartfile to the readme but forgot to remove the branch.
I tried integrating Navigation with Carthage a standalone application and it downloaded Navigation, Directions and Polyline correctly.